fancy-settings/source/settings.js

```diff
@@ -11,7 +11,7 @@ function localMessageHandler(msg, port) {
 
 // Test for the presence of an Edit Server
 function test_server() {
-	var port = chrome.extension.connect();
+	var port = browser.runtime.connect();
```

Comment:
Hmm, https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/runtime/connect says it's supported but apparently not :(